src/documentation/0024-npm-semantic-versioning/index.md

```diff
@@ -39,7 +39,7 @@ The rules use those symbols:
 
 Let's see those rules in detail:
 
-* `^`: if you write `^0.13.0` when running `npm update` it can update to patch and minor releases: `0.13.1`, `0.14.0` and so on.
+* `^`: if you write `^0.13.0`, when running `npm update` it can update to patch releases: `0.13.1`, `0.13.2` and so on(previous to 1.0.0, it only updates patch versions). If you write `^1.13.0`, when running `npm update` it can update to patch and minor releases: `1.13.1`, `1.14.0` and so on.
```

Comment:
```suggestion
* `^`:  It will only do updates that do not change the leftmost non-zero number. If you write `^0.13.0`, when running `npm update`, it can update to `0.13.1`, `0.13.2`, and so on, but not to `0.14.0` or above. If you write `^1.13.0`, when running `npm update`, it can update to `1.13.1`, `1.14.0` and so on, but will not update to `2.0.0` or above.
```